CEPCPreamble.md

```diff
@@ -0,0 +1,34 @@
+
+It should be every participant's goal to respect the code, their fellow participants, and to create a congenial and welcoming environment for all. 
+
+It can be helpful to call attention to certain features of the code from time to time so that we are aware of what it includes. There is a lot of nuance in this code, and it is worth reading through the whole thing: https://www.w3.org/Consortium/cepc/.
+
+The code encourages participants to: 
+* Have empathy for others, particularly when discussing sensitive or complex issues; we do not always know what others have experienced and should avoid making light of serious issues 
+* Be inclusive and promote diversity of people, perspectives, and experiences; encourage new participants and voices to join the conversation
+* Be respectful of others' needs and requests; when in doubt, ask and respect the response
+
+The code outlines the following unacceptable behaviours: 
+* Offensive comments relating to any factor of human difference, such as race, religion, gender, disability, body, or age
+* Misgendering or deliberately using a name or pronouns that a person has asked not to be used 
+* Invading the personal space or boundaries of another person after being asked to stop 
+* Microaggressions, which are comments or questions that marginalize people by communicating hostile, derogatory, or negative beliefs 
+
+This Code prioritizes the safety of individuals, particularly those in marginalized communities, over the comfort of others. This Code also prioritizes the impact of a statement or action over its intent. This means that while a person may not have intended to offend, their comments or actions may cause harm. When this occurs, we should do our best to respect the needs of the people affected, even if it can make some uncomfortable. By embracing discomfort, we can all grow. 
+
+We all make mistakes, what matters is how we address them. Together we can make W3C a positive and inclusive environment for all. 
+
+## Short Version: 
+
+The W3C's Code of Ethics and Professional Conduct sets out to create a positive environment for W3C in which all participants feel safe and respected. It documents behaviors that are expected as well as behaviors that are unacceptable. There is a lot of nuance in this code, and it is worth reading through the whole thing: https://www.w3.org/Consortium/cepc/.
+
+This Code prioritizes the safety of individuals, particularly those in marginalized communities, over the comfort of others. This Code also prioritizes the impact of a statement or action over its intent. This means that while a person may not have intended to offend, their comments or actions may cause harm. When this occurs, we should do our best to respect the needs of the people affected, even if it can make some uncomfortable. By embracing discomfort, we can all grow. 

```

Comment:
s/This means that while/While/

I'm not sure that discomfort itself is what makes us all grow.  I suggest dropping the last sentence.